src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#"
```

Comment:
The `href` attribute for the navigation links is set to `#`, which means they don't lead anywhere. Consider providing meaningful URLs for these links to improve navigation.